### DISCUSSION TITLE
Can I configure the travel speed?

### DISCUSSION THREAD

**Discussion by DolevCP** [vlachoudis/bCNC#1709] [Q&A]
Hello,

I started looking for how to custom configure bCNC for my needs and I didn't find any information regarding configuring the travel speed.
I managed to find how to configure specific steps besides the default ones, and now I would like to know if there is a way to customize the travel speed.

Thank you

**Comment by Harvie** [comment]
Maximum (rapid) travel speed is configured directly in GRBL
Cutting speed is part of the G-Code

Both of these can be configured "through bCNC", but they are not really something that bCNC has to take care of.

**Reply by DolevCP** [reply]
Can you tell me how can I configure the cutting speed?
If we're both talking about the same thing, which is the speed in which the machine is moving when cutting in the material.

**Reply by daxliniere** [reply]
This is determined in your gcode, as generated by your CAM software (such as Fusion 360, etc)

**Comment by MARIOBASZ** [comment]
You can see G code explanación in the web
G0 ;G1 ;G2 ;G3 ; F. 

g0 x200 y20 z40
G1 x109 y35 F1200

**Comment by MARIOBASZ** [comment]
When cutting on the path to be milled, bCNC sets the cutting speed according to the stock / material tabs.
It is the current way, although to choose correctly it should also take into account the material and number of edges of the end mill, cutting strategy
